esp/esdllib/esdl_transformer2.cpp

```diff
@@ -1511,6 +1511,18 @@ int Esdl2Transformer::process(IEspContext &ctx, EsdlProcessMode mode, const char
     if (!mi)
         throw MakeStringException(-1, "ESDL - method '%s::%s'not found", service, method);
 
+    // check the auth_feature
```

Comment:
are the auth_features declared in the request structure, or the method structure?